### PR TITLE
WIP: Change TreehouseUIKitView's size as necessary

### PR DESCRIPTION
**Pull request by swankjesse**
---

- [ ] `CHANGELOG.md`'s "Unreleased" section has been updated, if applicable.

### CODE REVIEW

**Comment by swankjesse** [comment]
https://github.com/cashapp/redwood/assets/133019/b53ff938-b7e5-4618-bef8-972ed0f26be2

File: redwood-treehouse-host/src/iosMain/kotlin/app/cash/redwood/treehouse/TreehouseUIView.kt [hunk 1]
```diff
@@ -60,13 +67,73 @@ public class TreehouseUIView private constructor(
...
+     */
     override fun layoutSubviews() {
+      super.layoutSubviews()
+
+      val oldWidthThatFits = sizeThatFits.useContents { width }
+      val oldHeightThatFits = sizeThatFits.useContents { height }
+
+      // TODO(jessewilson): is passing the screen bounds to sizeThatFits() appropriate?
+      this.sizeThatFits = sizeThatFits(
+        UIScreen.mainScreen.bounds.useContents<CGRect, CValue<CGSize>> {
```
Comment (JakeWharton) [inline redwood-treehouse-host/src/iosMain/kotlin/app/cash/redwood/treehouse/TreehouseUIView.kt:94]: Seems questionable. Root views are not guaranteed to occupy a full screen. Presumably they could also render on other screens.

Comment (swankjesse) [reply redwood-treehouse-host/src/iosMain/kotlin/app/cash/redwood/treehouse/TreehouseUIView.kt:94]: Yep, agreed.

**Comment by swankjesse** [comment]
Obsoleted by https://github.com/cashapp/redwood/pull/1858

**Pull request closed without merging**